src/Microsoft.ML.Data/Data/IDataLoader.cs

```diff
@@ -97,7 +97,8 @@ public interface IDataTransform : IDataView, ICanSaveModel
     /// Data transforms need to be able to apply themselves to a different input IDataView.
     /// This interface allows them to implement custom rebinding logic.
```

Comment:
>logic [](start = 65, length = 5)

Until we can delete this altogether, it may be worthwhile to add a note here to explain that the preferred idioms is that things work via the new `ITransformer` mechanism, which is the successor of this sort of idiom. I might not suggest adding the `[Obsolete]` tag -- that's a bit heaviweight -- just something in the XML docstring here like, "yeah, yeah, this is here, but we're trying to get rid of it, don't write new code with it."